gen3/bin/db.sh

```diff
@@ -562,6 +562,143 @@ gen3_db_restore() {
+  fi
+
+  # Quick check to ensure the snapshots were taken successfully. Will prevent new db from getting incomplete data.
+  echo "Did the snapshots get created correctly?(yes/no)"
+  read snapshotBool
+
+  if [[ $snapshotBool != "yes" ]]; then
+    gen3_log_err "Snapshots were not indicated to be taken correctly. Please clean up the old ones and try again"
+    gen3_log_err "If you ran out of space please setup a new volume and use the third parameter to this command to specify a different snapshot directory"
+    exit 1
+  fi
+
+  # Switch to the main commons terraform to grab current config then create new terraform and use original config with added parameters
+  gen3 workon $account $profile
+  gen3 cd
+  configFile=$(cat config.tfvars)
+  gen3 workon $account "$profile"__encrypted-rds
+  gen3 cd
+  mv config.tfvars config.tfvars-backup
+  echo "$configFile">>config.tfvars
+  echo "security_group_local_id=\"$securityGroupId\"" >> config.tfvars
+  echo "aws_db_subnet_group_name=\"$dbSubnet\"" >> config.tfvars
+  echo "db_pg_name=\"$dbParameterGroupName\"" >> config.tfvars
+  gen3 tfplan
+  gen3 tfapply
+
+  # Use sed to update all secrets, remove the arborist and metadata g3auto folders to recreate those db's then run kube-setup-secrets 
+  local newFenceDbUrl=$(aws rds describe-db-instances --filters '{"Name": "db-instance-id", "Values": ["'$vpc_name'-encrypted-fencedb"]}' | jq -r .DBInstances[0].Endpoint.Address)
+  local newIndexdDbUrl=$(aws rds describe-db-instances --filters '{"Name": "db-instance-id", "Values": ["'$vpc_name'-encrypted-indexddb"]}' | jq -r .DBInstances[0].Endpoint.Address)
+  local newGdcApiDbUrl=$(aws rds describe-db-instances --filters '{"Name": "db-instance-id", "Values": ["'$vpc_name'-encrypted-gdcapidb"]}' | jq -r .DBInstances[0].Endpoint.Address)
+  local originalFenceDbUrl=$(aws rds describe-db-instances --filters '{"Name": "db-instance-id", "Values": ["'$vpc_name'-fencedb"]}' | jq -r .DBInstances[0].Endpoint.Address)
+  local originalIndexdDbUrl=$(aws rds describe-db-instances --filters '{"Name": "db-instance-id", "Values": ["'$vpc_name'-indexddb"]}' | jq -r .DBInstances[0].Endpoint.Address)
+  local originalGdcApiDbUrl=$(aws rds describe-db-instances --filters '{"Name": "db-instance-id", "Values": ["'$vpc_name'-gdcapidb"]}' | jq -r .DBInstances[0].Endpoint.Address)
+  gen3_log_info "Updating fence db name from $originalFenceDbUrl to $newFenceDbUrl in $(gen3_secrets_folder)"
+  grep -rl $originalFenceDbUrl "$(gen3_secrets_folder)" | xargs sed -i "s/$originalFenceDbUrl/$newFenceDbUrl/g"
+  grep -rl $originalIndexdDbUrl "$(gen3_secrets_folder)" | xargs sed -i "s/$originalIndexdDbUrl/$newIndexdDbUrl/g"
+  grep -rl $originalGdcApiDbUrl "$(gen3_secrets_folder)" | xargs sed -i "s/$originalGdcApiDbUrl/$newGdcApiDbUrl/g"
+
+  gen3_log_info "Disabling gitops sync before updating secrets to ensure services not rolled during db setup"
+  kubectl delete cronjob gitops-sync
```

Comment:
use `g3kubectl` in scripts - it shouldn't make a difference now, but it used to - might save us some trouble later